src/localization/en/learn.json

```diff
@@ -150,7 +150,7 @@
   "steps.flagsMultiline.description": "Regex sees all text as one line. But we use the `multiline` flag to handle each line separately. In this way, the expressions we write according to the end of the linework separately for each line. Now enable the `multiline` flag to find all matches.",
 
   "steps.flagsCaseInsensitive.title": "Case-insensitive Flag",
-  "steps.flagsCaseInsensitive.description": "In order to remove the case-sensitivity of the expression we have written, we must activate the `case-insensitive` flag.",
+  "steps.flagsCaseInsensitive.description": "In order to remove the case-sensitiveness of the expression we have written, we must activate the `case-insensitive` flag.",
```

Comment:
Some changes you previously made in this merge request have been revert.

Reply:
oh! i understand the distinction now! it read weird to me at first, but now i see why "sensitiveness" is the proper choice for here.